package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "rsschool-codejam",
+  "version": "1.0.0",
+  "description": "Implementation of some functions in JS",
+  "main": "test.js",
+  "scripts": {
+    "test": "mocha"
+  },
```

Comment:
better to add one more script to use eslint check:

`"lint": "eslint src/*.js"`